chain/ethereum/ethereum.go

```diff
@@ -162,7 +162,6 @@ func ChunkedFilterLogs(ctx context.Context, client ETHClient, addresses []common
 			log.Error().Err(err).Msg("hit RPC rate limit")
 			return nil, err
 		}
-		log.Error().Err(err).Uint64("from", fromBlock).Uint64("to", toBlock).Msg("skipping failed backfill interval...")
```

Comment:
can you change it to `warn` and keep? it is good to know why it fails.

Reply:
This log is redundant since since the error is already returned to the caller. The caller will log it one more time.

Reply:
fixed